main.go

```diff
@@ -658,14 +669,17 @@ func decodeGMO(ctx context.Context, protoData *ProtoData, scanParameters decoder
 	if scanParameters.ProcessCells {
 		decoder.UpdateClientMapS2CellBatch(ctx, dbDetails, newMapCells)
 		if scanParameters.ProcessGyms || scanParameters.ProcessPokestops {
-			if !(len(newMapPokemon) == 0 && len(newNearbyPokemon) == 0 && len(newForts) == 0) {
-				decoder.ClearRemovedForts(ctx, dbDetails, newMapCells)
-			}
+			decoder.ClearRemovedForts(ctx, dbDetails, newMapCells)
```

Comment:
This could be in a goroutine with a separate context as it seems that this is a more expensive operation where people are seeing GMO timeouts?